Cargo.toml

```diff
@@ -31,3 +31,8 @@ macros = ["ethnum-macros"]
 ethnum-intrinsics = { version = "=1.1.0", path = "intrinsics", optional = true }
 ethnum-macros = { version = "=1.1.0", path = "macros", optional = true }
 serde = { version = "1", default-features = false, optional = true }
+
+[dev-dependencies]
+serde  = { version = "1", features = ["derive"] }
+serde_json = "1"
+bincode = "1"
```

Comment:
Would it be possible to keep these dependencies out of the test (my motivation is mostly to make building and `cargo test`-ing `ethnum` crate as lean as possible). Especially since the `serde_json` with pre-compiled binary fiasco, I would prefer to keep it out.

You should be able to use `Vec<u8> as IntoDeserializer` to check the `visit_seq` implementation.

Reply:
Feel free to take a stab at this (otherwise, I don't mind doing it myself on top of these changes).